src/sentry/api/endpoints/organization_member/team_details.py

```diff
@@ -83,7 +85,7 @@ def _is_org_owner_or_manager(access: Access) -> bool:
 
 @region_silo_endpoint
 class OrganizationMemberTeamDetailsEndpoint(OrganizationMemberEndpoint):
-    permission_classes = [TeamOrgMemberPermission]
+    permission_classes = [OrganizationTeamMemberPermission]
```

Comment:
these are usually tuples using round brackets - is there a reason for the square ones?

Reply:
I think it was like and I didn't notice, I'll change it to a tuple